src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterResponseDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/character")
+@RequiredArgsConstructor
+@Tag(name = "Character API", description = "API for managing characters")
+public class CharacterController {
+    private final CharacterService characterService;
+
+    @GetMapping("/random")
+    @Operation(summary = "Get a random character",
+            description = "Retrieve a random character from the database.")
+    public CharacterResponseDto getRandom() {
+        return characterService.getRandomCharacter();
+    }
+
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```
about why we removed it - it's more RESTful, names should be resource oriented